.github/workflows/build.yml

```diff
@@ -240,28 +242,49 @@ jobs:
         with:
           repository: ${{ github.repository }}
 
-      - name: Generate the CHANGELOG
-        uses: RiskLedger/generate-changelog@v1.2
-        id: changelog
+      - name: Configure Git
+        run: |
+          git config user.name "kingmakerbot"
+          git config user.email "kingmakerbot@users.noreply.github.com"
+
+      # automatic helm repo management
+      # (from https://github.com/liqotech/liqo/blob/master/.github/workflows/integration.yml)
+      - name: Install Helm
+        uses: azure/setup-helm@v3
         with:
-          from: ${{ steps.last-release.outputs.release }}
-          to: ${{ needs.configure.outputs.version }}
-        env:
-          GITHUB_AUTH: ${{ secrets.CI_TOKEN }}
+          version: v3.9.4
+
+      - name: Download chart releaser
+        run: |
+          curl -sSL "https://github.com/helm/chart-releaser/releases/download/v1.4.0/chart-releaser_1.4.0_linux_amd64.tar.gz" | tar -xz
 
-      - name: Save the CHANGELOG as a file
+      - name: Package helm chart
         run: |
-          echo "${{ steps.changelog.outputs.changelog }}" > ./CHANGELOG.md
-          sed -i "1s/.*/## Changes since ${{ steps.last-release.outputs.release }}/" ./CHANGELOG.md
+          # the output should be in the .cr-release-packages since cr index expects to find it there to create the helm index
+          helm dependency update deploy/crownlabs && helm package deploy/crownlabs --version "${{ needs.configure.outputs.version }}" --app-version "${{ needs.configure.outputs.version }}" --destination .cr-release-packages
 
-      - name: Create the release
-        uses: actions/create-release@v1
+      - uses: ncipollo/release-action@v1
         with:
-          tag_name: ${{ needs.configure.outputs.version }}
-          release_name: Release ${{ needs.configure.outputs.version }}
-          body_path: ./CHANGELOG.md
-        env:
-          GITHUB_TOKEN: ${{ secrets.CI_TOKEN }}
+          artifacts: ".cr-release-packages/*"
+          generateReleaseNotes: true
```

Comment:
Given you are using the automatic release notes generation, you should delete the `lerna.json` file and create a `.github/release.yml` file, as done [here](https://github.com/liqotech/liqo/blob/master/.github/release.yml).